modules/display.js

```diff
@@ -0,0 +1,35 @@
+/* eslint-disable */
+export default class UI {
+  static displayBooks() {
+    const books = store.getBooks();
```

Comment:
- [ ] I am able to add and remove books but when I add books and reload the page they are gone. Kindly ensure that when a page is reloaded the added books in the `localStorage` are present. I think it may have to do with this undefined `store` error here.

  ![ddu](https://user-images.githubusercontent.com/36542986/202130726-21efc81c-f5d5-4d64-9d86-104829c3b19f.jpg)